g2p/mappings/utils.py

```diff
@@ -751,9 +758,7 @@ def check_mapping_types(self) -> "_MappingModelDefinition":
             and self.rules_path is None
         ):
             LOGGER.warning(
-                exceptions.MalformedMapping(
-                    "You have to either specify some rules or a path to a file containing rules."
-                )
+                "Empty mapping: specify some rules or a path to a file containing rules."
```

Comment:
The exception we used to use here printed a lot of irrelevant information, instead, here is a narrow message with just the relevant details.